src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -953,6 +953,10 @@ sap.ui.define([
 		}
 		if (!vServerValue && oRequestInfo.method === HTTPMethod.PUT) {
 			this._setProperty(this.oDataServerState, aResPath, FHIRUtils.deepClone(oResource), true);
+		} else if (vServerValue && oRequestInfo.method === HTTPMethod.PUT && deepEqual(vServerValue, oResource)) {
+			// special handling when the server data and the client data before applying the change is the same(after multiple reset changes)
```

Comment:
add a whitespace before bracket. `is the same (after`

Reply:
done!